torch/csrc/jit/frontend/sugared_value.h

```diff
@@ -772,5 +772,29 @@ struct TORCH_API SugaredEnumClass : public SugaredValue {
   EnumTypePtr enum_type_;
 };
 
+struct TORCH_API PythonSliceValue : public SugaredValue {
```

Comment:
Maybe `SliceValue` because this is a regular `SugaredValue`?

Reply:
This is actually something I want to get advice on. It is a vanilla `SugaredValue` only because `ir_emitter` needs it and `ir_emitter` can't depend on `python_sugared_value`. In reality, `PythonSliceValue` does represent a python object, thus the `python` prefix. Let me know what you think.

Reply:
Hm, there is a `ClassValue` class that also extends `SugaredValue` that represents Python classes. So I think `SliceValue` would be better here.

Reply:
Done.